astrocut/cube_cut.py

```diff
@@ -88,7 +88,21 @@ def _parse_table_info(self, table_header, table_row):
             One row from the cube image header data table.
         """
 
-        # Turning the table row inot a new header object
+        data_ind = int(len(table_data)/2)  # using the middle file for table info
+        table_row = None
+
+        # Making sure we have a row with wcs info
+        while table_row is None:
```

Comment:
May want to add a check in case no table row is found

Reply:
Done.